lib/Backends/Habana/Habana.cpp

```diff
@@ -1011,6 +1027,18 @@ HabanaBackend::compile(Function *F, const BackendOptions &opts) const {
       convParams.emplace_back(std::move(params));
       break;
     }
+    case Kinded::Kind::LocalResponseNormalizationNodeKind: {
+      auto *NI = llvm::cast<LocalResponseNormalizationNode>(&I);
+      std::unique_ptr<ns_LrnKernel::Params> params = makeLrnParams(
+          NI->getAlpha(), NI->getBeta(), NI->getK(), NI->getHalfWindowSize());
+
+      chk(synCreateGenericNode(&tensors[NI->getInput()].get(),
+                               &tensors[NI].get(), 1, 1, (void *)params.get(),
+                               "lrn_f32", NI->getName().str().c_str(), nullptr,
```

Comment:
Is the datatype intentionally hardcoded? Does this operator not exist for other datatypes? It makes sense that it wouldn't, but I just want to confirm.

Reply:
Good eye - LRN does indeed only exist for f32.  We could assert that here, although it'll already be caught by isOpSupported.